docs/spl-docs/timechart-command.md

```diff
@@ -0,0 +1,105 @@
+# timechart
+
+## Description
+
+Creates a time series chart with corresponding table of statistics.
+
+A timechart is a statistical aggregation applied to a field to produce a chart, with time used as the X-axis. You can specify a split-by field, where each distinct value of the split-by field becomes a series in the chart. With the `limit` you can specify series filtering. If you set limit=0, no series filtering occurs.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+timechart \
+[limit=\<chart-limit-opt>] \
+[\<bin-options>... ] \
+(**\<single-agg>** [BY **\<split-by-clause>**] )
+
+## Required Arguments
+
+When specifying `timechart` command arguments, `<single-agg>`  BY `<split-by-clause>` is required.
+
+### single-agg
+
+**Syntax:** `count or \<stats-func>(\<field>)` \
+**Description:** A single aggregation applied to a single field, including an evaluated field. For \<stats-func>, see [Stats function options](link-for-stats-page.md). No wildcards are allowed. The field must be specified, except when using the count function, which applies to events as a whole.
+
+### split-by-clause
+
+**Syntax:** `<field>` \
+**Description:** Specifies a field to split the results by.
+
+## Optional Arguments
+
+### bin-options
+
+**Syntax:** `span=<int>[<timescale>]`
+**Descripton:** Option that can be used to specify discrete bins, or groups, to organize the information. A span of each bin, based on time. If the timescale is provided, this is used as a time range. If not, this is an absolute bin length.
```

Comment:
I think you meant for `Syntax` and `Description` to be on separate lines. You might need a blank line, or to end the `Syntax` line with `\`
<img width="971" alt="image" src="https://github.com/user-attachments/assets/94abbb9e-1cc4-4267-aa8c-dadbed7685e4">

Same comment for lines 41-42